docs/open-api/tags.yaml

```diff
@@ -212,7 +212,7 @@ paths:
     post:
       tags:
         - tag
-      summary: Associate tags with metadata object
+      summary: Associate tags with metadata object, the supported metadata objects are CATALOG, SCHEMA, TABLE, FILESET, TOPIC, COLUMN
```

Comment:
The summary is too long as a title (the web openAPI plugin will uses the summary as catalog title)

Reply:
@mchades can you please check again?